lib/puppet/type/logical_volume.rb

```diff
@@ -35,6 +35,27 @@
         raise ArgumentError , "#{value} is not a valid logical volume size"
       end
     end
+    def insync?(is)
+        lvm_size_units = { "K" => 1, "M" => 1024, "G" => 1048576, "T" => 1073741824, "P" => 1099511627776, "E" => 1125899906842624 }
```

Comment:
By convention all code is two spaces indents.

For clarity, please use `1024*1024`, etc, instead of 1048576.